pgpool2-4.6.yaml

```diff
@@ -63,6 +68,45 @@ subpackages:
       runtime:
         - ${{package.name}}
 
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for Upstream pgpool image"
+    dependencies:
+      provides:
+        - pgpool2-compat=${{package.full-version}}
+      runtime:
+        - bash
+        - coreutils
+        - busybox
+        - postgresql
+        - postgresql-client
+        - postgresql-dev
+    pipeline:
+      - name: "Symlink all pgpool binaries to /opt/pgpool-II/bin"
+        runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/pgpool-II/bin
+          for bin in \
+            pcp_attach_node pcp_detach_node pcp_health_check_stats pcp_node_count \
+            pcp_node_info pcp_pool_status pcp_proc_count pcp_proc_info pcp_promote_node \
+            pcp_recovery_node pcp_stop_pgpool pcp_watchdog_info \
+            pg_enc pg_md5 pgpool pgpool_setup pgproto watchdog_setup wd_cli \
+          ; do
+            ln -s "/usr/bin/${bin}" "${{targets.subpkgdir}}/opt/pgpool-II/bin/${bin}"
+          done
+
+          # copy etc to pgpool/etc
+          mkdir -p "${{targets.contextdir}}/opt/pgpool-II/etc"
+          cp -pr "${{targets.destdir}}/etc/pgpool2/." "${{targets.contextdir}}/opt/pgpool-II/etc/"
+
+          # copy etc to pgpool/etc.default
+          mkdir -p "${{targets.contextdir}}/opt/pgpool-II/etc.default"
+          cp -pr "${{targets.destdir}}/etc/pgpool2/." "${{targets.contextdir}}/opt/pgpool-II/etc.default"
+      - name: "Fetch and add startup scripts"
+        runs: |
+          # Add start.sh and entrypoint scripts same as upstream
+          git clone --depth 1 --branch master https://github.com/pgpool/pgpool2_on_k8s
+          install -Dm755 pgpool2_on_k8s/pgpool.docker/start.sh  ${{targets.contextdir}}/opt/pgpool-II/bin/start.sh
+          install -Dm755 pgpool2_on_k8s/pgpool.docker/entrypoint.sh ${{targets.contextdir}}/opt/pgpool-II/bin/entrypoint.sh
+
```

Comment:
we should move this to a separate package and call it something like `${{package.name}}-oci-entrypoint` if it's entrypoint script. 

One package should ideally have only one source checkout.

Reply:
Noted, moving this to separate subpackage